### ISSUE TITLE
`locale` feature is not actually `locale`

### ISSUE THREAD

**Issue by grawk** [paypal/nemo-view#33]
See here:
https://github.com/paypal/nemo-view/pull/32

Unfortunately this feature is a carry-over and people do use it, with the property `data.locale`. So while it would be good to rename the feature to "specialization" (since that's essentially what it is) we cannot remove the existing feature. Also using a `data.*` config property might not be the best way to deal with specialization.

Another option would be to expose `get/set` methods for modifying specialization on the fly. E.g.

`nemo.view.setSpecialization('Germany')`
`nemo.view.getSpecialization()`

Thus we don't ask people to define a data property.

Also, while rectifying this issue, we need to honor backward compatibility for some time in order to allow people to change their impelementations.

**Comment by OuranosSkia** [comment]
I had actually been meaning to comment on this for a while. What is implemented in that PR sounds more like a "switch" than anything. On a personal level I can't see why a locator (or any code level object) would change names with locale, but I can see a lot of use in allowing people to specify switches in their locators. Making it more generic might expand its use case and get better feedback on how people would want to be using it.

**Comment by grawk** [comment]
You're absolutely right! The feature should be called "switch". Still wondering about implementation though. What do you think about set and get methods for the current switch value?

**Comment by OuranosSkia** [comment]
Yeah, will definitely need get/set. Will want to be able to change it while running tests. I'm not super sure what I would call the variable that controls this though... "switch" seems odd for a variable name. Maybe "flow" or something like that. I would imagine a typical use case to be setting it for a whole test, so you'd really be doing a flow path for that test.
